tox.ini

```diff
@@ -68,6 +68,40 @@ commands =
 	 # file.  See pyproject.toml for legal <change type> values.
 	 python -m towncrier.check
 
+[testenv:draftnews]
+basepython=python3.6
+passenv = TAHOE_LAFS_* PIP_* SUBUNITREPORTER_* USERPROFILE HOMEDRIVE HOMEPATH
+# Get "certifi" to avoid bug #2913. Basically if a `setup_requires=...` causes
+# a package to be installed (with setuptools) then it'll fail on certain
+# platforms (travis's OX-X 10.12, Slackware 14.2) because PyPI's TLS
+# requirements (TLS >= 1.2) are incompatible with the old TLS clients
+# available to those systems.  Installing it ahead of time (with pip) avoids
+# this problem.
+whitelist_externals = mv
+deps =
+     certifi
+     towncrier
+skip_install = true
+commands =
+         # With pip >= 10 the existence of pyproject.toml (which we are
+         # required to have to configure towncrier) triggers a "build
+         # isolation" mode which prevents anything from working.  Avoid
+         # triggering that pip behavior by keeping the towncrier configuration
+         # somewhere else and only bringing it in when it's actually needed
+         # (after pip is done).
+         #
+         # Some discussion is available at
+         # https://github.com/pypa/pip/issues/5696
+         mv towncrier.pyproject.toml pyproject.toml
+
+         # this burns in another copy of the version -- but we can't
+         # import tahoe until it supports python3.5+
+         python -m towncrier --draft --version 1.14.0
```

Comment:
Should we put a totally and completely bogus version string here instead?  `--version 100.100.100` or something - or do you expect this string should be updated with each Tahoe-LAFS release?

Or maybe this only has to stick around until we get a new towncrier release with Python 2.x support.